libvncclient/rfbproto.c

```diff
@@ -1780,6 +1850,15 @@ SendClientCutText(rfbClient* client, char *str, int len)
 
   if (!SupportsClient2Server(client, rfbClientCutText)) return TRUE;
 
+  if ((!str) || (len < 0)) {
```

Comment:
While a noble check, does this really relate to the feature? If not, please remove it and let's add it in later :-)

Reply:
it is lib api...I can remove it...

Reply:
yeah please do so